web/packages/teleport/src/Navigation/ResourcesSection.tsx

```diff
@@ -215,6 +220,17 @@ function getResourcesSubsections({
       onClick: () => setPinnedUserPreference(false),
       subCategory: CustomNavigationSubcategory.FilteredViews,
     },
+    {
+      title: 'Git Servers',
```

Comment:
I think we should be consistent with the types filter panel:
<img width="250" alt="Screenshot 2025-02-03 at 9 17 09 AM" src="https://github.com/user-attachments/assets/5caa9294-6bb9-4005-9fc3-3982bc4068cd" />

Currently it's called just `Git`:
https://github.com/gravitational/teleport/blob/2004893be0a8314a4a2b9307a4490dfd7baa48ba/web/packages/shared/components/UnifiedResources/FilterPanel.tsx#L40-L48

I am ok updating that one to `Git server` or keep this one `Git`,  as long as they are consistent. I am bad at naming. I think some other alternatives are `Git Platforms`, `Git Hosting`, IDK.

Reply:
`tsh git ls` says list Git servers so seems like it would be Git server.

Reply:
cool. do you mind if i push the filter panel renaming change here?

Reply:
sure, sounds good

Reply:
looks like the filter panel use singular so i kept it `Git Server`. interestingly name for SSH doesn't match either.